providers/scaleway/flags.go

```diff
@@ -0,0 +1,87 @@
+package scaleway
+
+import (
+	"os"
+
+	"github.com/scaleway/scaleway-sdk-go/scw"
+	"github.com/urfave/cli/v2"
+)
+
+const category = "Scaleway"
+
+//nolint:mnd
+var ProviderFlags = []cli.Flag{
+	&cli.StringFlag{
+		Name:    "scw-access-key",
```

Comment:
Should we use `scaleway` instead of `scw`?

Reply:
No strong opinion, `scaleway` is fine for me as well.